experiments/dataSources/HubSpotREST/extendSkuidDB/README.md

```diff
@@ -0,0 +1,37 @@
+# Extend HubSpot with SkuidDB
+This is only available when using Skuid NLX. This sample page shows how you can store and track scoring of an in-progress deal over time in SkuidDB. 
+
+## Features
+* Query single record using URL parameter
+* Connect to a table in SkuidDB to store deal scoring based on SPICED methodology
+* Save the deal score back to HubSpot CRM as a custom field on the Deal object
+
+## Instructions for Skuid page
+- Page API:  V2
+- Data source: 
+1. REST data source named "HubSpotTestApp" connecting to your HubSpot instance per [instruction](HubSpotREST)
+2. Skuid database named "Demos", with a table named "HubSpot_SPICED_Score"
```

Comment:
I know this was your data source name,  but I'd recommend using something like "Hubspot_AddOns" as DB name here.